### ISSUE TITLE
Amounts not decoded (displayed) for "Decode outputs/Prove sending" 

### ISSUE THREAD

**Issue by mbg033** [graft-project/graft-blockchain-explorer#9]
```
(2020-05-11 18:05:10) [INFO    ] Request: 127.0.0.1:40040 0x7fcab0000bc0 HTTP/1.0 POST /myoutputs
Unsupported rct type: 

show_my_outputs: Cant decode RingCT!
Unsupported rct type: 
show_my_outputs: key images: Cant decode RingCT!
Unsupported rct type: 
show_my_outputs: key images: Cant decode RingCT!
(2020-05-11 18:05:10) [INFO    ] Response: 0x7fcab0000bc0 /myoutputs 200 0

```

**Comment by mbg033** [comment]
```
(2020-05-11 18:05:10) [INFO    ] Request: 127.0.0.1:40040 0x7fcab0000bc0 HTTP/1.0 POST /myoutputs
Unsupported rct type: 

show_my_outputs: Cant decode RingCT!
Unsupported rct type: 
show_my_outputs: key images: Cant decode RingCT!
Unsupported rct type: 
show_my_outputs: key images: Cant decode RingCT!
(2020-05-11 18:05:10) [INFO    ] Response: 0x7fcab0000bc0 /myoutputs 200 0

```

**Comment by mbg033** [comment]
fix cherry-picked from upstream repo ab07b3e0e6487c0702642b8d2232bd9f4df91808
